comp/core/configsync/configsyncimpl/sync.go

```diff
@@ -10,12 +10,57 @@ import (
 	"encoding/json"
 	"net/http"
 	"reflect"
+	"strconv"
 	"time"
 
 	apiutil "github.com/DataDog/datadog-agent/pkg/api/util"
 	pkgconfigmodel "github.com/DataDog/datadog-agent/pkg/config/model"
 )
 
+func (cs *configSync) updater() {
+	cfg, err := fetchConfig(cs.ctx, cs.client, cs.Authtoken.Get(), cs.url.String())
+	if err != nil {
+		if cs.connected {
+			cs.Log.Warnf("Failed to fetch config from core agent: %v", err)
+			cs.connected = false
+		} else {
+			cs.Log.Debugf("Failed to fetch config from core agent: %v", err)
+		}
+		return
+	}
+
+	if cs.connected {
+		cs.Log.Debug("Succeeded to fetch config from core agent")
+	} else {
+		cs.Log.Info("Succeeded to fetch config from core agent")
+		cs.connected = true
+	}
```

Comment:
```suggestion
        cs.Log.Debug("Succeeded to fetch config from core agent")
	if !cs.connected {
	    cs.connected = true
	}
```

Reply:
They have different log levels for a reason; so we can track the first connection always. Is this a problem?

Reply:
Ohh I did not see the log levels. My bad